README.md

```diff
@@ -39,7 +39,31 @@ setup separate feature branches
 
 ### Branches
 
-Branch names clearly communicate purpose and should contain a reference to issue tracking system if relevant
+***Basic Rules***
+
+**1. Lowercase and hyphen-separated:** Keep branch names in lowercase and separate words with hyphens. For example, use `feature/booking-tool` or `bugfix/cookie-banner-refresh`.
+
+**2. Alphanumeric characters:** Use only alphanumeric characters (letters a-z, numbers 0–9) and hyphens. Avoid using punctuation, spaces, underscores, or any other non-alphanumeric characters.
```

Comment:
I would prefer to make it even more strict, not avoid but do not use. And for fun add the regex (feature|bugfix|hotfix|docs|chore)\/[a-z0-9]+(-([a-z0-9])+)*[a-z0-9]+